README.md

```diff
@@ -0,0 +1,24 @@
+# Moddingway
+
+Discord moderation bot for NAUR.
+
+## How To Run
+Run `docker compose up --build` after providing the appropriate environment variables listed below in the `.env` file.
+
+### Release
+#### Environment variables:
+- DISCORD_TOKEN
+- POSTGRES_USER
+- POSTGRES_PASSWORD
+
+### Testing
+#### Environment variables:
+- DISCORD_TOKEN
+- POSTGRES_USER
+- POSTGRES_PASSWORD
```

Comment:
i haven't been paying too much attention to the POSTGRES stuff. are these two vars storing a pre-created user/password or are we configuring it with the environment vars?

Reply:
Just saw this, we're configuring it with environment vars since we removed it in #27

Reply:
let's specify that in the readme